src/pages/compose/index.tsx

```diff
@@ -34,11 +35,20 @@ const Compose = () => {
   const [deleteModalVisible, setDeleteModalVisible] = useState(false);
 
   const dispatch = useDispatch();
-  const handleSaveModalOk = () => { 
+  const handleSaveModalOk = async () => { 
```

Comment:
Rename this function as handleSaveProjectInfo.
Function name needs to be meaning full always.